app/register/page.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { test, expect } from '@playwright/test';
+
+const user = {
+  confirmPassword: 'test12345',
+  email: 'test1@email.com',
+  incorrectEmail: 'test@email.com',
+  incorrectPassword: 'test',
+  invalidconfirmPassword: 'tester',
+  invalidEmail: 'test',
+  invalidPassword: 'tester',
+  password: 'test12345',
+};
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('/register');
+});
+
+test.describe('Tests register page', () => {
+  test('should successfully register and direct users to weekly picks', async ({
+    page,
+  }) => {
+    await page.getByTestId('email').fill(user.email);
+    await page.getByTestId('password').fill(user.password);
+    await page.getByTestId('confirm-password').fill(user.confirmPassword);
+    await page.getByTestId('continue-button').click();
+    await page.goto('/weeklyPicks');
```

Comment:
Re: line 26
It's sabotaging the test. We want Playwright to find its way to the weekly picks page because we used the correct credentials and clicked the button, not because we told it where to go.